guides/06_FAQs.md

```diff
@@ -0,0 +1,239 @@
+
+The URIs resolve to IP addresses in the AMAZON subnets for us-east-1, as published in the [AWS ip-ranges.json](https://docs.aws.amazon.com/general/latest/gr/aws-ip-ranges.html).
+
+### What are the subnets and ports used by a client running the Amazon Chime SDK?
+
+A client running the Amazon Chime SDK connects to the Amazon Chime service via URIs (never direct IP addresses) that are part of the *.chime.aws domain.  The URIs resolve to IP addresses in the CHIME_MEETINGS subnets, as published in the [AWS ip-ranges.json](https://docs.aws.amazon.com/general/latest/gr/aws-ip-ranges.html), which is currently a single subnet: 99.77.128.0/18.
+
+The Amazon Chime SDK will use TCP:443 for https and websocket connectivity, and UDP:3478 for media.  If UDP:3478 is blocked, the SDK falls back to TCP, however the video experience will be poor, and audio will use a region specific subnet assigned by [Amazon Elastic Compute Cloud (Amazon EC2)](https://aws.amazon.com/ec2/).
+
+### Can a client running the Amazon Chime SDK connect to the service via a proxy?
+
+The Amazon Chime SDK has URL[rewriter hooks](https://aws.github.io/amazon-chime-sdk-js/classes/meetingsessionurls.html#urlrewriter) that the application builder can use to munge the URLs to traverse unauthenticated proxies.  When traversing a proxy, all media is sent over TCP which impacts the user experience.
+
+## Mobile browser
+
+### I am unable to select audio output device in some browsers, is this a known issue?
+
+[Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1152401) and [Safari](https://bugs.webkit.org/show_bug.cgi?id=179415) have known issues disallowing them from listing audio output devices on these browsers. While clients can continue the meeting using the default device, they will not be able to select devices in meetings.
+
+### Is screen capture supported on mobile browsers?
+
+Amazon Chime JS SDK applications do not support content sharing on mobile browsers because a key dependency, WebRTC's [getDisplayMedia](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getDisplayMedia) is not available on these browsers. Clients joining from these devices can still view content shared by other participants in the call.
+
+### Will Amazon Chime JS SDK meetings work in Chrome and Firefox in iOS?
+
+At present, Amazon Chime JS SDK is supported on Safari/iOS only. Other browsers are missing a key dependency, [GetUserMedia](https://bugs.webkit.org/show_bug.cgi?id=208667).
+
+### When loading a webpage in iOS WebView, my application gets a TypeError: undefined is not an object (evaluating 'navigator.mediaDevices.addEventListener'), what could be the issue ?
+
+This is because Safari embedded [web view does not support WebRTC](https://forums.developer.apple.com/thread/88052). The mitigation is to use the Amazon Chime SDK for iOS to build a native application: https://github.com/aws/amazon-chime-sdk-ios
```

Comment:
https://forums.developer.apple.com/thread/88052  - the link is broken (403) as of 6:24 PM PST. Probably, a temporary error?

Reply:
yeah this used to work. will fix it later

Reply:
It works now Kyu - with no change from us